policyengine_us/parameters/gov/states/mt/tax/income/credits/elderly_homeowner_or_renter_credit/net_household_income/standard_exclusion.yaml

```diff
@@ -0,0 +1,20 @@
+description: Montana allows for this standard exclusion for the computation of the elderly homeowner or renter credit.
```

Comment:
```suggestion
description: Montana allows for this standard exclusion amount of net household income under the elderly homeowner or renter credit.
```

Reply:
right?

Reply:
Net household income = Max_((Gross household income - standard exclusion), 0) * rate. I think it's correct